stdlib/io/src/main/ballerina/src/io/readable_character_channel.bal

```diff
@@ -64,6 +64,26 @@ public type ReadableCharacterChannel object {
         return readXmlExtern(self);
     }
 
+# Reads a property from a .properties file.
+# ```ballerina
+# string|io:Error result = readableCharChannel.readProperty(key);
+# ```
+# + key - The property key needs to read.
+# + return - The read property value or else an `io:Error`
+    public function readProperty(string key) returns @tainted string|Error {
```

Comment:
Shall we include default value too? If not nil this will return the return value in case property is not set in the file.

Reply:
Enable return `()` when the key-value pair is not available. Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/24978/commits/69ad64013e0efec7c6976329696de8557797775c

Reply:
What I mean was to tweak the API as,
```
public function readProperty(string key, string defaultValue) returns @tainted string?|Error {
```

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-lang/pull/24978/commits/d2062dc7ebe28ab52dd7a33bc5fac8815876ab18